src/components/MatchList.js

```diff
@@ -0,0 +1,69 @@
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+
+import { fetchMatchList } from '../redux/fetchMatchList/fetchMatchList';
+import minimap from '../assets/Minimap.png';
+import styles from '../styles/MatchList.module.css';
+
+export default function Matchlist() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMatchList());
+  }, []);
+
+  const list = useSelector((state) => state.list);
+  let count = 1;
+
```

Comment:
- [OPTIONAL] It will be nice to implement a stateless component that filters the data retrieved from the API by some parameters. For example you can implement a search input that filters the matches displayed on the `Home page` based on the duration, match id, date...... Checkout the ***live demo*** of this [project](https://github.com/jerryowusu/covid-19-tracking-webapp/tree/dev) for inspiration.

Reply:
As this page relies on live content, sometimes match duration is not ready and, mostly, all matches retrieved are from today. Although I could implement some other way of sorting the list, thank you!